test/CadlRanchProjects.Tests/azure-core-basic.cs

```diff
@@ -44,46 +45,57 @@ public class AzureCoreBasicTests : CadlRanchTestBase
         public Task Azure_Core_Basic_list() => Test(async (host) =>
         {
             AsyncPageable<User> allPages = new BasicClient(host, null).GetUsersAsync(5, 10, null, new[] {"id"}, "id lt 10", new[] {"id", "orders", "etag"}, new[] {"orders"});
-            await foreach (Page<User> page in allPages.AsPages())
+            var results = new List<User>();
```

Comment:
I think we should test the pageable feature `AsPages()`, each page has Property `Values` which is a list of User. and just check the `Values` content.
The previous test is correct. I think.

Reply:
OK then I would like to close it